src/js/playground/playground.js

```diff
@@ -252,6 +252,13 @@ export class Playground {
     const blocks = this.workspace.topBlocks_;
     console.log(blocks);
     checkNextBlock(blocks);
+    if(blocks[0].colour_ !== blocks[0].style.colourPrimary){
```

Comment:
```suggestion
    if(blocks[0].color_ !== blocks[0].style.colorPrimary){
```

SUGGESTION: Up to now, we have always used US spelling. I'm not sure where using British spelling started, but I'm seeing it here for the first time. Using British spelling may seem no big deal, but in fact makes it difficult to search for certain words, such as the word color, and thereby also can make refactoring more difficult. I would rather have all spellings conform to the more common US spelling rules. Also, spell checkers may flag one or the other depending on one's settings, and that may confuse developers in changing it back and forth, or even result in ignoring spelling suggestions altogether as they occur too frequently and then tend to be dismissed.

If you change it here, also please do so at all possible places in the code. 

Are there any other instances of regional spelling variants that could be concerned? Here are some that may occur in the context of Catrobat: 

- flavor versus flavour: concerning our design flavors, which are reflected on share and in our apps, such as the Pocket Code versus the Luna and Cat flavors. 
- program versus programme: for code
- neighbor versus neighbour: which could occur w.r.t parsing of formula trees
- license versus licence: related to our FLOSS licenses
- dialog versus dialogue: in UI code

I'm sure there must be more.

Reply:
https://github.com/google/blockly/blob/master/blocks/colour.js
We are overriding variables from Blockly, so we can't prevent using colour instead of color.

Reply:
> https://github.com/google/blockly/blob/master/blocks/colour.js
> We are overriding variables from Blockly, so we can't prevent using colour instead of color.

Acknowledged. I just checked, it's a complete mixup in Blockly, so it's practically impossible to clean it up because both color and colour appear at too many places in the code, and most likely many third party projects depend on it already: github reports 354 search results of "colour" in 9 programming languages and 206 search results of "colour" in 7 programming languages for google/blockly.